src/core/scene/a-scene.js

```diff
@@ -317,6 +342,50 @@ module.exports.AScene = registerElement('a-scene', {
       writable: window.debug
     },
 
+    enableCanvasPointerLock: {
```

Comment:
What are the scenarios where someone would want to enable / disable this? I don't fully understand Edge WebVR behavior. I'm poking on this with partial information.

Reply:
Just wanted to be flexible. I can't think of a scenario that this would be useful. I will remove this if it's okay with you.

Reply:
As a maintainer I'm becoming more and more flag averse over time 😄 since new flags combine with previous ones that can combine in multiple weird ways that are difficult to debug and maintain over time. As a conservative measure I would get rid of the flag now and reintroduce it later if we found a legitimate use case for it. I cannot think of any at the moment.

Reply:
Sure, this makes sense, removing!